src/linear_algebra/dimension.lean

```diff
@@ -806,8 +809,7 @@ begin
     exact infinite_basis_le_maximal_linear_independent b v i m, }
 end
 
-theorem basis.mk_eq_rank'' {ι : Type v} (v : basis ι R M) :
-  #ι = module.rank R M :=
+theorem basis.mk_eq_rank' {ι : Type v} (v : basis ι R M) : #ι = module.rank R M :=
```

Comment:
Can you revert this rename (replacing it with a todo comment)? It will make the diff much smaller and therefore easier to forward-port.

Reply:
It didn't make the diff much smaller…